html-css/week1/index.html

```diff
@@ -4,30 +4,37 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <title>Bogdan Dobinda Front-end Developer½</title>
     <link rel="stylesheet" href="style.css" />
```

Comment:
You can add a <meta> tag for a description that would appear in search engine results. It would be something like this:
`<meta name="description" content="Bogdan Dobina, is an experienced Front-end Developer based in Copenhagen, specialized in HTML, CSS, JavaSript and React.">`

Reply:
I did not know how to optimize in SEO a website but now I have read it some info about it. One friend told me to do that for his website 1 month back and I told him that I have no idea how to optimize his website to appear in google search.  Good to know for the future. :)